Calling/ClientApp/src/core/constants.ts

```diff
@@ -8,8 +8,6 @@ export class Constants {
   static CROP_MEDIA = 'Crop';
   static CONFIGURATION_LOCAL_VIDEO_PREVIEW_ID = 'ConfigurationLocalVideoPreview';
   static LOCAL_VIDEO_PREVIEW_ID = 'LocalVideoPreview';
-  static MINI_HEADER_WINDOW_WIDTH = 360;
+  static MINI_HEADER_WINDOW_WIDTH = 680;
```

Comment:
how did we determine this should be 680?

Reply:
Synced offline, it is determined by whenever the window is too narrow to contain all the buttons